### PR TITLE
Multiple choice click to reveal

### PR DESCRIPTION
**Pull request by LucasDooms**
Add an option to hide all multiple choice options until you tap the screen.
If you turn on this option, it also shows a hint to tell you what the type of the answer should be (see screenshots).

Also separated the fragment_quiz.xml file in to two extra files (multiple choice part and keyboard part).

(please verify the French translations)

<div>

</div>

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -196,4 +196,11 @@
     <string name="delete_voices_files">Effacer les fichiers de voix</string>
     <string name="dont_ask_voices">Ne plus demander</string>
     <string name="voices_reinit_done">Fichiers de voix supprimés</string>
+    <string name="tap_to_reveal_choices">Appuyez pour révéler les choix</string>
+    <string name="tap_to_reveal_choices_summary">Masquez les options de choix multiples jusqu\'au moment où vous touchez l\'écran</string>
```
Comment (jehutyno) [inline app/src/main/res/values-fr/strings.xml:200]: Les choix sont cachés jusqu'à ce que vous touchiez l'écran

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -196,4 +196,11 @@
...
+    <string name="give_hiragana_reading_hint">Donnez la lecture en hiragana</string>
+    <string name="give_romaji_hint">Donnez le rōmaji</string>
```
Comment (jehutyno) [inline app/src/main/res/values-fr/strings.xml:202]: Donnez la lecture en rōmaji

**Review by jehutyno — CHANGES_REQUESTED** [review]
Small french translation changes.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (LucasDooms) [reply app/src/main/res/values-fr/strings.xml]: done!

**Pull request merged**